### DISCUSSION TITLE
Is it possible to send the email confirmation email after signup?

### DISCUSSION THREAD

**Discussion by sarukuku** [supabase/supabase#680] [Questions]
I'd like to login the user automatically after they've created their account. I can only do this if I disable email confirmations. I'd still like to provide the user a chance to confirm their email later. How could I send this confirmation email but not have it conditional for login?

**Comment by sarukuku** [comment]
@kiwicopple I know you guys must be super busy and there are a million things on your roadmap but even a hint on what you current thinking about this is would be nice 🙂

**Reply by silentworks** [reply]
I don't think this is possible at the moment, but hopefully in the future they will provide a way by allowing cloud functions or a webhook for events that happen on database tables.

**Reply by kiwicopple** [reply]
Hey yeah sorry I'm trying to get through as many of the GoTrue quirks as possible. 

For this one, this is what we are thinking: the [`POST /invite`](https://github.com/supabase/gotrue#post-invite) endpoint could return the invite URL. If you have emails disabled, you would just store this invite link and then send it to the user at a later date.

I'm tracking this one here: https://github.com/supabase/gotrue/issues/65
